shell/ci/release/docker.sh

```diff
@@ -164,6 +148,14 @@ build_and_push_image() {
         -t "$remote_image_name:$VERSION" -t "$remote_image_name:latest" --push \
         "$buildContext"
     )
+  else
+    # When we're not on a tag, just build the docker image to verify
+    # that it is buildable.
+    info "Building Test Docker Image"
```

Comment:
I don't love this, because it implies that the image is used for testing. "Running Test Docker Build" maybe? "Verifying Docker Build"?

Reply:
I'll address this in a follow-up. Hard agree. I didn't like the phrasing much either.